changelog.d/5-internal/specify-sending-to-deleted-legalhold-device

```diff
@@ -0,0 +1 @@
+Specify (in a test) how a message to a deleted legalhold device is refused to be sent.
```

Comment:
I would have gone for "improve test coverage for legalhold", but this is good, too.